src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -98,12 +98,28 @@ public int Main(string[] args)
                 throw;
             }
 #else
+            catch (AggregateException aex)
+            {
+                foreach (var exception in aex.InnerExceptions)
+                {
+                    DumpException(exception);
+                }
+                return 1;
+            }
             catch (Exception ex)
             {
-                AnsiConsole.GetError(useConsoleColor: _runtimeEnv.OperatingSystem == "Windows").WriteLine($"Error: {ex.Message}".Red().Bold());
+                DumpException(ex);
                 return 1;
             }
 #endif
         }
+
+        private void DumpException(Exception ex)
+        {
+            AnsiConsole
+                .GetError(useConsoleColor: _runtimeEnv.OperatingSystem == "Windows")
+                .WriteLine($"Error: {ex.Message}".Red().Bold());
```

Comment:
Because we only want to show the short summary message here. `ToString` includes the stack trace